client/me/purchases/billing-history/utils.jsx

```diff
@@ -82,6 +83,124 @@ export function renderTransactionAmount( transaction, { translate, addingTax = f
 	);
 }
 
+function renderTransactionQuantitySummaryForTitanMail(
+	licensed_quantity,
+	new_quantity,
+	isRenewal,
+	isUpgrade,
+	translate
+) {
+	if ( isRenewal ) {
+		return translate( 'Renewal for %(quantity)s mailbox', 'Renewal for %(quantity)s mailboxes', {
+			args: { quantity: licensed_quantity },
+			count: licensed_quantity,
+			comment: '%(quantity)s is number of mailboxes renewed',
+		} );
+	}
+
+	if ( isUpgrade ) {
+		return translate(
+			'Purchase of %(quantity)s new mailbox',
+			'Purchase of %(quantity)s new mailboxes',
```

Comment:
May I suggest changing this copy to 'Purchase of %(quantity)s additional mailboxes' and do that for all the variations when `isUpgrade` is `true`?

Reply:
I like that 👍.

Reply:
Done